packages/calcite-components/src/components/block/block.tsx

```diff
@@ -93,7 +93,7 @@ export class Block
   @Prop() heading!: string;
 
   /**
-   * Specifies the number at which section headings should start.
+   * Specifies the semantic heading level of the internally-rendered text container for proper document structure, without affecting visual styling.
```

Comment:
This is a bit hard to read and offers a lot of context for users to sift through. Is there a different approach we can take as an in-between for the previous language and what is proposed?

Potential suggestion below, but flexible on a different direction.

```suggestion
   * Specifies the heading level of the component's `heading` for proper document structure, without affecting visual styling.
```